src/components/auth/login/Login.jsx

```diff
@@ -1,41 +1,84 @@
-import React from "react";
+import React, { useState, useRef } from "react";
 import { auth } from "../../config/firebaseConfig";
 import db from "../../config/firebaseConfig";
 import AddProducts from "../../addProductsForm/AddProducts";
-import GoogleAndFacebookLogIn from "./GoogleAndFacebookLogIn";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
 
 const LogInForm = () => {
-  const [isAdmin, setIsAdmin] = React.useState(false);
+  const [isAdmin, setIsAdmin] = useState(false);
+  const [isLoggedIn, setLoggedIn] = useState(false);
+  const passwordValue = useRef(false);
+  const emailValue = useRef(false);
 
   const userLogout = () => {
     auth.signOut().then(() => {
       setIsAdmin(false);
+      setLoggedIn(false);
     });
   };
+
   const userLogin = async (e) => {
-    e.preventDefault();
-    const email = e.target[0].value;
-    const password = e.target[1].value;
-    const userLogin = await auth.signInWithEmailAndPassword(email, password);
+    const userLogin = await auth.signInWithEmailAndPassword(
+      emailValue,
+      passwordValue
+    );
+
     const loggedInUserName = await db
       .collection("users")
       .doc(userLogin.user.uid)
       .get();
     setIsAdmin(loggedInUserName.data().isAdmin);
+    setLoggedIn(true);
   };
 
+  const emailGroup = (
+    <Form.Group controlId="formGroupEmail">
+      <Form.Label>Email address</Form.Label>
+      <Form.Control type="email" placeholder="Enter email" ref={emailValue} />
+    </Form.Group>
+  );
+
+  const passwordGroup = (
+    <Form.Group controlId="formGroupPassword">
+      <Form.Label>Password</Form.Label>
+      <Form.Control
+        type="password"
+        placeholder="Password"
+        ref={passwordValue}
+      />
+      {isLoggedIn ? (
+        <Button
+          variant="info"
+          size="md"
+          className="loginButton"
```

Comment:
it should be possible to collapse this code more concisely. e.g., <Button> should only be seen once but onClick and the button text have ternary operators

Reply:
Yes, you are right!